src/assets/scss/modules/_rocket.scss

```diff
@@ -0,0 +1,40 @@
+@import './mixins';
+
+.rocket-container {
+  display: flex;
+
+  &__img {
+    width: 18%;
+  }
+}
+
+.rocket-info {
+  padding: 0 1rem;
+  @include flex-pos(column, start);
+  align-items: start;
```

Comment:
Hey Mike, I see you're using the mixins we made using scss, 
for the second parameter of this mixin, I believe you wanted to use "flex-start" instead of "start". we can check it out later today

Reply:
Hello @Zeraltz!
Both "flex-start" and "start" have the same effect, in this case, to keep the title, description, and button at the beginning of the box.
![image](https://user-images.githubusercontent.com/97407840/179846652-db872c20-e1ce-4964-a8e9-fc0bf8cc0150.png)